src/Containers/DistrictContainer/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import React from 'react';
+import DistrictCard from '../../Components/DistrictCard';
+import { getDistricts } from '../../utils/api';
+import type { DetailedDistrictType } from '../../types';
+
+export class DistrictContainer extends React.Component {
+  state: {
+    districts: Array<DetailedDistrictType>,
+  };
+
+  constructor() {
+    super();
+    this.state = { districts: [] };
```

Comment:
Only reason I have used a constructor so far is to transform props into state, but I rarely do that